go.mod

```diff
@@ -130,7 +129,10 @@ require (
 	sigs.k8s.io/cluster-api-provider-gcp v1.7.1-0.20240724153512-c3b8b533143c // indirect
 )
 
-require github.com/gophercloud/utils/v2 v2.0.0-20240701101423-2401526caee5
+require (
```

Comment:
any reason to sequester these in their own require block?

Reply:
`go mod tidy` did it -- I didn't touch the file by hand.